rdtools/test/soiling_test.py

```diff
@@ -50,8 +50,8 @@ def test_soiling_srr(soiling_normalized_daily, soiling_insolation, soiling_times
     expected_means = expected_means[['soiling_rate', 'soiling_rate_low', 'soiling_rate_high',
                                      'inferred_start_loss', 'inferred_end_loss',
                                      'length', 'valid']]
-    pd.testing.assert_series_equal(expected_means, soiling_info['soiling_interval_summary'].mean(),
-                                   check_exact=False, check_less_precise=6)
+    actual_means = soiling_info['soiling_interval_summary'][expected_means.index].mean()
+    pd.testing.assert_series_equal(expected_means, actual_means, check_exact=False)
```

Comment:
@kanderso-nrel Can you say a little more about this change?

Reply:
`soiling_info['soiling_interval_summary']` has some datetime columns that aren't present in `expected_means`.  Older versions of pandas silently dropped those columns when taking the mean, so it ended up working out that there wasn't any mismatch when calling `assert_series_equal`.  However new versions of pandas warn that datetime columns won't get dropped in a future version, so this change just explicitly drops them prior to taking the mean.  This serves both to silence the warning and to be future-proof for whenever pandas makes that change.  Here's an example of that warning on another PR:  https://github.com/NREL/rdtools/runs/2549105826?check_suite_focus=true#step:5:342

Reply:
That makes sense, thanks!